src/seedu/addressbook/ui/TextUi.java

```diff
@@ -18,65 +17,62 @@
  */
 public class TextUi {
 
-    /**
-     * A decorative prefix added to the beginning of lines printed by AddressBook
-     */
-    public static final String LINE_PREFIX = "|| ";
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
 
-    /**
-     * A platform independent line separator.
-     */
-    public static final String LS = System.lineSeparator();
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
 
-    public static final String DIVIDER = "===================================================";
+    private static final String DIVIDER = "===================================================";
```

Comment:
It's not just security. We should not expose a member unless it's needed by an external party. A bit like having glass walls in a home - we want to use them sparingly even if they are unbreakable glass.